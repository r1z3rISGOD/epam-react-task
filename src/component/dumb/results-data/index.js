import {ResultsData} from "./results-data"
export {ResultsData}
