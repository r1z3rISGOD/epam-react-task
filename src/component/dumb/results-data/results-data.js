"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.ResultsData = void 0;
var React = require("react");
var ResultsData = function (_a) {
    var children = _a.children;
    return (React.createElement("div", { className: "results-data" }, children));
};
exports.ResultsData = ResultsData;
//# sourceMappingURL=results-data.js.map