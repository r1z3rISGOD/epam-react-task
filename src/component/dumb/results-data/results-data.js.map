{"version":3,"file":"results-data.js","sourceRoot":"","sources":["results-data.tsx"],"names":[],"mappings":";;;AAAA,6BAA8B;AAMvB,IAAM,WAAW,GAAgC,UAAC,EAAY;QAAV,QAAQ,cAAA;IACjE,OAAO,CACD,6BAAK,SAAS,EAAC,cAAc,IACxB,QAAQ,CACP,CACX,CAAA;AACH,CAAC,CAAA;AANY,QAAA,WAAW,eAMvB"}