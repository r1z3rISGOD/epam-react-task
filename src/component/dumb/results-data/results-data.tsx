import * as React from 'react'

interface ResultsDataProps {
    children: React.ReactNode
}

export const ResultsData : React.FC<ResultsDataProps> = ({ children }) => {
  return (
        <div className="results-data">
            {children}
        </div>
  )
}
