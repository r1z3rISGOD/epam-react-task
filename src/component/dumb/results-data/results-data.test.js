import { ResultsData } from './results-data.tsx'
import React from 'react'

it('results data', () => {
  const comp = shallow(<ResultsData children={() => {}}/>)
  expect(comp).toMatchSnapshot()
})
