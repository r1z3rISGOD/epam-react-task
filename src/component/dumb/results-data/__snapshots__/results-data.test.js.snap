// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`results data 1`] = `
<div
  className="results-data"
>
  <Component />
</div>
`;
